test/client_options_test.py

```diff
@@ -226,12 +229,14 @@ def test_config_required_override(config, tmpdir):
         "-c", "required",
         "--password-file", str(password_file),
         "--cafile", "/engine2.pem",
+        "--insecure",
         "--log-file", "test.log",
         "--log-level", "debug",
     ])
     assert args.engine_url == "https://engine.com"
     assert args.username == "username"
     assert args.cafile == "/engine2.pem"
+    assert args.secure is False
     assert args.disk_timeout == 120
     assert args.log_file == "test.log"
     assert args.log_level == "debug"
```

Comment:
We need to test the case when the config file contains secure=False, unless we don't
want to support this, but this seems useful for development environment, for closed networks where people do not care about the certificates.